cime_config/testdefs/testmods_dirs/clm/midDecStart/include_user_mods

```diff
@@ -0,0 +1 @@
+../default
```

Comment:
Add README saying that midDecStart is used instead of decStart if you want the run split to happen in December rather than January (i.e., before rather than after new year).